docs/wiki/tutorials/taiga_client.md

```diff
@@ -1,80 +1,141 @@
 # Taiga client
 
-This client is used to interact with Taiga API.
-
 ## Initialization
 
 Using username and  password:
-> client = j.clients.taiga.new('test', host="https://staging.circles.threefold.me/", username='7mada', password='123456')
-
+```
```

Comment:
is 7amada really going to be part of the official documentation? :)

Reply:
updated to admin ^_^